strings/sillytips.txt

```diff
@@ -29,4 +29,5 @@ Absolutely nothing unusual will happen this shift. Promise.
 Send help. Trapped in tip writing factory.
 There is no such thing as a cow level.
 HONK.
-Solars probably gives radiation poisoning or something, considering the other power sources are deadly in their own ways.
\ No newline at end of file
+Solars probably gives radiation poisoning or something, considering the other power sources are deadly in their own ways.
+Some people are unable to read text on a game where half of it is based on text.
```

Comment:
This is already in the code on line 21 as @Lunaimon pointed out

Reply:
I can't help but feel like this is ironic